components/feature/prompts/src/main/java/mozilla/components/feature/prompts/PromptFeature.kt

```diff
@@ -140,6 +142,7 @@ class PromptFeature private constructor(
     private val onManageLogins: () -> Unit = {},
     private val creditCardPickerView: SelectablePromptView<CreditCard>? = null,
     private val onManageCreditCards: () -> Unit = {},
+    private val onSelectCreditCard: () -> Boolean = { true },
```

Comment:
I noticed there is a explosion of functions related to logins, we could add a delegate similar to [ShareDelegate](https://github.com/mozilla-mobile/android-components/blob/master/components/feature/prompts/src/main/java/mozilla/components/feature/prompts/share/ShareDelegate.kt), this can be addressed in a follow up, please lets file a ticket for it.
<img width="1019" alt="image" src="https://user-images.githubusercontent.com/773158/119992418-86a7a500-bf98-11eb-8a97-068d9a953d73.png">

Reply:
Should we indicate in the doc, what is the usages of the `Boolean` returned by `onSelectCreditCard`?